### PR TITLE
Initial stab at generating some integration files we need to further …

### PR DESCRIPTION
**Pull request by jtpetty**
…automate the release process

### CODE REVIEW

File: src/Misc/InstallAgentPackage.template.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (damccorm) [inline src/Misc/InstallAgentPackage.template.xml]: It feels like these probably belong under the `.azure-pipelines` directory, right? They're not really src code

File: src/Misc/UnpublishVSTSAgent.template.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = 'Stop'
```
Comment (damccorm) [inline src/Misc/UnpublishVSTSAgent.template.ps1]: We probably don't actually need this file anymore if we're going to only roll forward. Fine leaving it if you feel like we should have it for the moment though.

File: src/Misc/InstallAgentPackage.template.xml (hunk 1 above)
Comment (jtpetty) [reply src/Misc/InstallAgentPackage.template.xml]: I'm not sure anything under Misc is really "source code". I don't have a strong preference where they live.

File: src/Misc/UnpublishVSTSAgent.template.ps1 (hunk 2 above)
Comment (jtpetty) [reply src/Misc/UnpublishVSTSAgent.template.ps1]: I think we should leave it until we have a roll forward process defined.

File: src/Misc/InstallAgentPackage.template.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <StepData>
+        <AddTaskPackageData packageType="agent" platform="win-x64" version="<AGENT_VERSION>" downloadUrl="https://vstsagentpackage.azureedge.net/agent/<AGENT_VERSION>/vsts-agent-win-x64-<AGENT_VERSION>.zip" infoUrl="https://go.microsoft.com/fwlink/?LinkId=798199" filename="vsts-agent-win-x64-<AGENT_VERSION>.zip" />
+      </StepData>
+    </ServicingStep>
+    <ServicingStep name="Add x86 Windows agent package" stepPerformer="DistributedTask" stepType="AddTaskPackage">
+      <StepData>
+        <AddTaskPackageData packageType="agent" platform="win-x86" version="<AGENT_VERSION>" downloadUrl="https://vstsagentpackage.azureedge.net/agent/<AGENT_VERSION>/vsts-agent-win-x86-<AGENT_VERSION>.zip" infoUrl="https://go.microsoft.com/fwlink/?LinkId=798199" filename="vsts-agent-win-x86-<AGENT_VERSION>.zip" />
+      </StepData>
+    </ServicingStep>
+    <ServicingStep name="Add x64 Redhat 6 agent package" stepPerformer="DistributedTask" stepType="AddTaskPackage">
```
Comment (vtbassmatt) [inline src/Misc/InstallAgentPackage.template.xml]: "Add x64 RHEL6 agent package" or "Add x64 Red Hat Enterprise Linux 6 agent package".

**Comment by jtpetty** [comment]
I am going to go ahead and push in these changes, but will follow up with updates from the review comments.

**Pull request merged**